content/en/aws/sns/index.md

```diff
@@ -75,3 +75,75 @@ LocalStack Pro users can access our [LocalStack App's](https://app.localstack.cl
    $ awslocal sns list-topics
    {{< /command >}}
 5. Use the web user-interface to perform further operations on the SNS topic, such as deleting the topic, visualizing the topic attributes, and creating a new subscription.
+
+## Localstack Specifics
+
+### Accessing published Platform Messages
+
```

Comment:
Can you please add a line or two here what LocalStack specifics mean in this context?